api/tests/exceptions/test_exception_handlers.py

```diff
@@ -23,6 +24,23 @@ def push_log_raising():
     assert response.status_code == 400
     assert response.json == {'code': 400, 'message': 'Validation error'}
 
+def test_csrf_error_exception_handler(client, app, monkeypatch):
+
+    def sacct_raising():
+        raise CSRFError('CSRF Error')
+
+    monkeypatch.setitem(app.view_functions, 'sacct_', sacct_raising)
+    response = client.post('/manager/sacct')
```

Comment:
issue: let's try to avoid binding tests to specific endpoints, when reasonable. This way we don't have to update this test in case that endpoint changes or is removed

In this case you are testing the handler, I would call the handler directly with a CSRFError and assert on the returned response

Reply:
I like the proposal, although I would have to refactor all the endpoints (and I wouldn't do it in this PR)
I'll change the current one and take notes!